packages/cli/src/commands/source/push/delete.ts

```diff
@@ -0,0 +1,165 @@
+import {Command, flags} from '@oclif/command';
+import {Source} from '@coveo/push-api-client';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+import {
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import dedent from 'ts-dedent';
+import {
+  buildAnalyticsFailureHook,
+  buildAnalyticsSuccessHook,
+} from '../../../hooks/analytics/analytics';
+import {green, red} from 'chalk';
+// eslint-disable-next-line node/no-extraneous-import
+import {AxiosResponse} from 'axios';
+
+interface ErrorFromAPI {
+  response: {
+    status: number;
+    data: {
+      errorCode: string;
+      message: string;
+    };
+  };
+}
+
+export default class SourcePushDelete extends Command {
+  public static description =
+    'Delete one or multiple documents in a given push source. See https://docs.coveo.com/en/171 and https://docs.coveo.com/en/131';
```

Comment:
Let's check with @fbeaudoincoveo , I reckon he likes those `<>` around URIs :P

Reply:
Haha yea

I'll open another PR to review this documentation :)

Reply:
@fbeaudoincoveo : You can just comment on this PR and I'll process it.